<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthorDialog</class>
 <widget class="QDialog" name="AuthorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QFrame" name="OptionsFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>661</width>
     <height>171</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="DisplayLabelFormatOptions">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Input Text Formatting Options</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="FnLn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Firstname-Lastname (i.e. John Smith)</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="DisplayLableAuthorNameOrder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Author Name Order</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="LnFn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Lastname-Firstname (i.e. Smith John)</string>
    </property>
   </widget>
   <widget class="QLabel" name="DisplayLabelSeparatorsSection">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Separators:</string>
    </property>
   </widget>
   <widget class="QLabel" name="DisplayLabelFnLnSeparator">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>First Name/Last Name Separator:</string>
    </property>
   </widget>
   <widget class="QLabel" name="DisplayLabelEntrySeparator">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Entry Separator (between authors):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FnLnSeparator">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>\s</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EntrySeparator">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>80</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>\t</string>
    </property>
   </widget>
   <widget class="QLabel" name="DisplayLabelSanitizerOptions">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sanitizer Options</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="EnableSanitizer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable Sanitizer</string>
    </property>
   </widget>
   <widget class="QLabel" name="DisplayLabelG1Regex">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Group 1 Regex</string>
    </property>
   </widget>
   <widget class="QLabel" name="DisplayLabelG2Regex">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Group 2 Regex</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="G1Regex">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>[\w+\.]</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="G2Regex">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>140</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>[A-ZÀ-ß]</string>
    </property>
   </widget>
   <widget class="QLabel" name="DisplayLabelSpacingCharacter">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>120</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Spacing Character (\s for space, \t for tab):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SpacingChar">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>140</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>\t</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>\s for space,\t for tab</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="EditFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>661</width>
     <height>301</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="DisplayLabelRawText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Raw Text</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="SourceText">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>471</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="OutputText">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>471</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(243, 243, 243);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string>[output text]</string>
    </property>
   </widget>
   <widget class="QLabel" name="DisplayLabelOutput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Output</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
